XPNet.GraphicsTest/Properties/PublishProfiles/FolderProfile.pubxml.user

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
.user files should generally be added to .gitignore instead of checked in.

Reply:
Removed properties folder, because it is not needed